doc/introduction.xml

```diff
@@ -477,43 +477,17 @@ SET client_min_messages TO notice;
 		</para>
 		<para>
-			Several changes were necessary with respect to MobilityDB version 1.0 to enable the above. An important one was the definition of new data types <varname>span</varname> and <varname>spanset</varname>, which provide similar functionality to the PostgreSQL data types <varname>range</varname> and <varname>multirange</varname> but can also be used in various programming languages regardless of PostgreSQL. These are <emphasis>template types</emphasis>, which means that they are containers of other types, in a similar way as lists and arrays contains values of other types. Furthermore, a new template data type <varname>set</varname> was also added. Therefore, the types <varname>timestampset</varname>, <varname>period</varname>, and <varname>periodset</varname> in version 1.0 are replaced by the types <varname>tstzset</varname>, <varname>tstzspan</varname>, and <varname>tstzspanset</varname> in version 1.1. The name of the constructor functions for these types was modified accordingly.
+			In order to enable the above, a refactoring of the code base was necessary. All the types provided by version 1.2 in addition to the new types <varname>tgeometry</varname> and <varname>tgeography</varname> are defined as core types and thus, are always included in the building process. Each one of the other spatiotemporal types can be included in the building process by setting its corresponding compilation flag. For example, when adding <varname>-DCBUFFER=1</varname> the types built upon the <varname>cbuffer</varname> base type, that is <varname>cbuffer</varname>, <varname>cbufferset</varname>, and <varname>tcbuffer</varname>, will be included in the building process. The possibility to select the types to be included in the executable program is important in particular for stream processing, due to the limited capability of the edge devices. Please notice that the new types <varname>tcbuffer</varname>, <varname>tpose</varname>, and <varname>trgeometry</varname> are still in development and will be finalized in a forthcoming release.
 		</para>
 		<para>
-			Finally, the API of MEOS and MobilityDB was streamlined to improve usability. We detail next the most important changes in the API.
+			In addition, the API of MobilityDB and MEOS was extended with new functionality and streamlined to improve usability. Finally, version 1.3 also enables the latest versions PostgreSQL 18 and PostGIS 3.6.0. For all these reasons, the binary format of the temporal types have changed from version 1.2 to 1.3 and thus a backup and restore is needed for migrating to the newer version 1.3 of MobilityDB.
```

Comment:
I'm merging this commit already, but did we really change the binary format of the existing types, like tgeompoint? What changed between 1.2 and 1.3?